client/error.py

```diff
@@ -159,16 +159,35 @@ class TaintConfigurationError:
     path: Optional[Path]
     description: str
     code: int
+    start_line: Optional[int]
+    start_column: Optional[int]
+    stop_line: Optional[int]
+    stop_column: Optional[int]
 
     @staticmethod
     def from_json(error_json: Dict[str, Any]) -> "TaintConfigurationError":
         try:
+            error_location = error_json["location"]
+            if error_location is not None:
```

Comment:
Just to confirm: does that mean that we export something like `"location": null` when we don't have a location? That's a bit odd, I would just expect to not have the key.

Reply:
This is doable :)